omero/users/cli/import-bulk.rst

```diff
@@ -0,0 +1,99 @@
+Using bulk imports
```

Comment:
**Using**  ? bulk imports. 

Maybe better simply ``Bulk imports`` 
there might be a false-friend problem with ``Using import targets`` - but this is a different case. Import targets are truly a valid description of a cli option(s) whereas ``bulk imports``  has much wider meaning.

Reply:
I'm going to push a commit changing "Using import targets" to just "Import targets". My primary goal was consistently in the index.